src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -149,7 +149,7 @@ public Task<string> StartNewAsync(string orchestratorFunctionName, object input)
         /// </summary>
         /// <param name="instanceId">The ID of the orchestration instance to query.</param>
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
-        public async Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId)
+        public virtual async Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId)
```

Comment:
Let's leave the implementation of `GetStatusAsync` here but create a corresponding abstract method in the abstract base class.